packages/donor/src/state/AppointmentToBookStore.ts

```diff
@@ -30,4 +32,19 @@ export class AppointmentToBookStore {
     this.donationStartTimeMillis = slot.donationStartTimeMillis;
     this.appointmentIds = slot.appointmentIds;
   }
+
+  isDonationValidForDonor(donor: Donor): boolean {
```

Comment:
Name is too generic.
How about `isAppointmentTooCloseToLastDonation`?

Reply:
Also, does this currently return true when donation is not valid?

Reply:
1. Like
2. I see what you mean, there us some mismatch between the the names and meanings, will fix